app/views/v1/subdomain_util.py

```diff
@@ -0,0 +1,109 @@
+from typing import Set, Tuple
+
+import json
+
+import flask_jwt_extended
+from flask import request, url_for
+
+import app.utils.authentication_utils as authentication_utils
+import app.actions as actions
+import app.db_models as db_models
+
+from . import bp
+from app.utils.ct_search import get_subdomains_from_ct
+from app.actions.add_targets import add_targets
+
+from time import time
+
+BATCH_SIZE = 10  # set appropriate amount
+RESCAN_INTERVAL = 62400  # to check target once a day
+
+
+@bp.route("/rescan_subdomains", methods=["GET"])  # is ok?
+@flask_jwt_extended.jwt_required
+def api_cron_rescan_subdomains() -> None:
+    rescan_subdomains()
+
+
+def rescan_subdomains() -> int:
+    targets_to_rescan = (
+        db_models.db.session.query(db_models.SubdomainRescanTarget)
+        .order_by(db_models.SubdomainRescanTarget.subdomain_last_scan.asc())
+        .limit(BATCH_SIZE).all()
+    )
+    for target in targets_to_rescan:
+        if time() - target.subdomain_last_scan < RESCAN_INTERVAL:
```

Comment:
The filtering should be directly in the SQL query, so that we don't have to order all targets.